backend/server/src/main/java/com/issuetracker/domain/NewIssue.java

```diff
@@ -0,0 +1,55 @@
+package com.issuetracker.domain;
+
+import java.util.Arrays;
+
+public class NewIssue {
+
+    private final String title;
+
+    private final String comment;
+
+    private final Long[] assigneesId;
+
+    private final Long[] labelsId;
```

Comment:
```suggestion
    private final Users assignees;

    private final Labels labels;
```

 이렇게 할 수는 없을지 제안해봅니다.

Reply:
도메인에서는 가능한 일급 컬렉션을 사용해주시고,
dto 에서는 List 로 통일 부탁드립니다.
컨벤션상으로 일반 Array 는 최대한 사용을 자제하도록 합시다.
이펙티브 자바 스타일입니다.

Reply:
https://github.com/ghojeong/issue-tracker/commit/c78c6164429820f4256bf80ea420fa76c630d4c3
해결완료 했습니다!